test/suites/pod5/hg001_giab_2023.05.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source "${TEST_UTILS_DIR}/utils.sh"
+
+  if ! aws s3 cp help &> /dev/null
+  then
+    echo "command 'cp' could not be found (possible solution: run 'ml awscli' before executing this script)"
+    exit 1
+  fi
+
+aws s3 cp --region eu-west-1 --no-sign-request s3://ont-open-data/giab_2023.05/flowcells/hg001/20230505_1857_1B_PAO99309_94e07fab/pod5_pass/ ${TEST_RESOURCES_DIR}/downloads/ --recursive --exclude PAO99309_pass__94e07fab_c3641428_* --include PAO99309_pass__94e07fab_c3641428_9*
+
+args=()
+args+=("--workflow" "pod5")
+args+=("--input" "${TEST_RESOURCES_DIR}/hg001_giab_2023_05.tsv")
+args+=("--config" "${TEST_RESOURCES_DIR}/hg001_giab_2023_05.cfg")
+args+=("--output" "${OUTPUT_DIR}")
+args+=("--resume")
+
+vip "${args[@]}" 1> /dev/null
+
+compare expected to actual output and store result
+if [ "$(zcat "${OUTPUT_DIR}/vip.vcf.gz" | grep -vc "^#")" -gt 0 ]; then
+  result="0"
```

Comment:
the output report contains 1 record and is 5.4MB, this might indicate an issue for larger number of records. possible cause: the genome browser displays many "white" reads possibly related to " In IGV, reads with a mapping quality = 0 are displayed in white while reads with a positive mapping quality are displayed in grey". 
![image](https://github.com/molgenis/vip/assets/2477186/8b988be2-5d9f-4721-bc0d-4aa382cc1084)